packages/date-picker/test/keyboard-input.test.js

```diff
@@ -702,22 +702,6 @@ describe('keyboard', () => {
         datePicker._focus();
       });
 
-      it('should change after validate on clear button click', () => {
-        datePicker.clearButtonVisible = true;
-        datePicker.$.clearButton.click();
-        expect(validateSpy.calledOnce).to.be.true;
-        expect(changeSpy.calledOnce).to.be.true;
-        expect(changeSpy.calledAfter(validateSpy)).to.be.true;
-      });
-
-      it('should change after validate on Esc with clear button', async () => {
```

Comment:
**note:** It's enough to verify that validation happens on clear button click. To verify that the clear button logic is invoked on Escape, I added a separate test: `should clear the value on Escape`.